public/robots.txt

```diff
@@ -1,3 +0,0 @@
-# https://www.robotstxt.org/robotstxt.html
```

Comment:
You may need this file to protect your site against being abused or overloaded with requests. Maybe search the importance of a robot file, for example: https://www.qed42.com/insights/coe/quality-assurance/importance-of-robots.txt

You don't have to have it though, you can delete it, but will be great if you just know this info for the future.